src/usdb_syncer/gui/usdb_upload_dialog.py

```diff
@@ -0,0 +1,108 @@
+"""Dialog to manage USDB uploads."""
+
+from __future__ import annotations
+
+from PySide6.QtWidgets import QDialog, QWidget
+
+from usdb_syncer.gui import progress, theme
+from usdb_syncer.gui.forms.UsdbUploadDialog import Ui_Dialog
+from usdb_syncer.gui.resources import styles
+from usdb_syncer.gui.theme import DiffPalette, _rgb_str
```

Comment:
Private import (wonder why ruff allows that). Maybe move generate_diff_css into theme.py.